storybook/pages/InDropdownPage.qml

```diff
@@ -2,7 +2,7 @@ import QtQuick 2.14
 import QtQuick.Controls 2.14
 import QtQuick.Layouts 1.14
 
-import AppLayouts.Chat.controls.community 1.0
+import AppLayouts.Communities.controls 1.0
```

Comment:
I wonder if `AppLayouts.Communities.popups 1.0` is not a better location for that. Bc it's also kind of popup.

Reply:
Do you think then all these ones should be also moved to **popups**?

![Screenshot 2023-06-26 at 14 47 41](https://github.com/status-im/status-desktop/assets/97019400/6b411a86-20ee-4b86-b4e9-f0d229ae8d3f)

Reply:
I'll do this reorganization in a different task:  https://github.com/status-im/status-desktop/issues/11265